src/main/java/uk/gov/justice/laa/crime/cfecrime/utils/RequestHandlerUtil.java

```diff
@@ -0,0 +1,55 @@
+package uk.gov.justice.laa.crime.cfecrime.utils;
+
+import lombok.experimental.UtilityClass;
+import lombok.extern.slf4j.Slf4j;
+import uk.gov.justice.laa.crime.cfecrime.api.Result.Outcome;
+import uk.gov.justice.laa.crime.cfecrime.api.*;
+
+@Slf4j
+@UtilityClass
+public class RequestHandlerUtil {
+
+    public static CfeCrimeResponse handleRequest(CfeCrimeRequest request){
+        CfeCrimeResponse response = new CfeCrimeResponse();
+        Boolean under18 = false;
+        Boolean passported = false;
```

Comment:
we can't assume these are false if not supplied. That's why they are Boolean rather than boolean